toonz/sources/toonz/preferencespopup.cpp

```diff
@@ -1755,6 +1765,14 @@ PreferencesPopup::PreferencesPopup()
         ioGridLay->addWidget(new QLabel(tr("FFmpeg Timeout:")), 4, 0,
                              Qt::AlignRight);
         ioGridLay->addWidget(m_ffmpegTimeout, 4, 1, 1, 3);
+        ioGridLay->addWidget(new QLabel(" "), 5, 0);
+        ioGridLay->addWidget(
+            new QLabel(tr("Please indicate where you would like "
+                          "exports from Fast Render(MP4) to go.")),
+            6, 0, 1, 4);
+        ioGridLay->addWidget(new QLabel(tr("Quick Render Path: ")), 7, 0,
```

Comment:
IMO the term associated with this feature should be standardized to "Fast Render" or "Quick Render".